### PR TITLE
Add mime types to GetImagePartType extension method

### LINKED ISSUE #1488: New ImagePart media added with extension .bin
**Describe the bug**

This is a follow up to an original bug https://github.com/dotnet/Open-XML-SDK/issues/1305 _"New ImagePart is created as .bin instead of real extension"_ which is still an issue with the most recent 2.20.0 release.

When adding a new `ImagePart` the media in the bundled package has an extension of `.bin` instead of `.jpg` etc.

**Screenshots**

**To Reproduce**

Create a new document and add an `ImagePart` using the following code:

```csharp
// Load an image and get the appropriate mime type
var image = System.Drawing.Image.LoadFromFile("myphoto.jpg");
var codes = ImageCodecInfo.GetImageDecoders();
var mimeType = codes.First(c => c.FormatID == image.RawFormat.Guid).MimeType;

var imagePart = mainPart.AddImagePart(mimeType); // image/jpeg

using var stream = new MemoryStream();
image.Save(stream, image.RawFormat);
stream.Position = 0;
imagePart.FeedData(stream);
```

**Observed behavior**
The media is added to the word document package with .bin extension

**Expected behavior**
The media is added to the word document package with .jpg extension

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Office version: 2306 - 16529.20154
 - .NET Target: .NET Core 6
 - DocumentFormat.OpenXml Version: 2.20.0

### PR DESCRIPTION
**Pull request by mikeebowen**
Fixes #1488

### CODE REVIEW

File: src/DocumentFormat.OpenXml/Packaging/ImagePartTypeInfo.cs [hunk 1]
```diff
@@ -62,6 +62,16 @@ internal static ImagePartType GetImagePartType(string extension)
                 ".svg" => ImagePartType.Svg,
                 ".tiff" => ImagePartType.Tiff,
                 ".wmf" => ImagePartType.Wmf,
+                "image/bmp" => ImagePartType.Bmp,
```
Comment (tomjebo) [inline src/DocumentFormat.OpenXml/Packaging/ImagePartTypeInfo.cs:65]: clever. probably should rename the input argument from "extension" to something more general (?). Not sure what.

Comment (tomjebo) [reply src/DocumentFormat.OpenXml/Packaging/ImagePartTypeInfo.cs:65]: Also, not sure if this is a good idea. Might be better to have a separate GetImagePartTypeByMime() or something.

Comment (twsouthwick) [reply src/DocumentFormat.OpenXml/Packaging/ImagePartTypeInfo.cs:65]: This seems fine - the API name isn't saying it's from extensions. How are we handling this in v3.0?

**Comment by twsouthwick** [comment]
Can you clean up the title and add `fixes #1488` to the body of the message?

File: src/DocumentFormat.OpenXml/Packaging/ImagePartTypeInfo.cs (hunk 1 above)
Comment (twsouthwick) [reply src/DocumentFormat.OpenXml/Packaging/ImagePartTypeInfo.cs:65]: > should rename the input argument

Probably ok, but that can be a source breaking change if someone did `GetImagePartType(extension: ".something")`

Comment (mikeebowen) [reply src/DocumentFormat.OpenXml/Packaging/ImagePartTypeInfo.cs:65]: > Having one method that handles this seems fine - the API name isn't saying it's from extensions. How are we handling this in v3.0?

v3.0 has the same issue, but the fix will have to be a bit different

**Review by twsouthwick — APPROVED** [review]

**Pull request merged**